ee/server/service/hosts.go

```diff
@@ -23,144 +19,3 @@ func (svc *Service) HostByIdentifier(ctx context.Context, identifier string, opt
 	opts.IncludePolicies = true
 	return svc.Service.HostByIdentifier(ctx, identifier, opts)
 }
-
-func (svc *Service) RunHostScript(ctx context.Context, request *fleet.HostScriptRequestPayload, waitForResult time.Duration) (*fleet.HostScriptResult, error) {
```

Comment:
THis is just a bit of refactoring to move script-related code to a `scripts.go` file instead of `hosts.go` which was already quite big.